storage/replica_raftstorage.go

```diff
@@ -112,16 +114,32 @@ func (r *Replica) Entries(lo, hi, maxBytes uint64) ([]raftpb.Entry, error) {
 	// If neither the number of entries nor the size limitations had an
 	// effect, we weren't able to supply everything the client wanted.
 	if len(ents) != int(hi-lo) && (maxBytes == 0 || size < maxBytes) {
```

Comment:
@mrtracy All we really have here, is the interface for the function:
https://github.com/coreos/etcd/blob/master/raft/storage.go